docs/integrations/positioning-systems/ublox-evk-m8n.mdx

```diff
@@ -0,0 +1,267 @@
+---
+title: UBlox EVK-M8N GPS
+sidebar_label: UBlox EVK-M8N GPS
+keywords:
+  - ublox
+  - gps
+  - leo
+  - integration
+  - example
+description: >-
+  Explore the steps to gather GPS data from a UBlox GPS module and integrate it
+  with your Leo Rover system using the UBlox EVK-M8N GPS module example.
+image: /img/robots/leo/integrations/ublox-gps/ublox-evk-m8n-shop.webp
+---
+
+import ProductPreview from '@site/src/components/ProductPreview';
+
+This tutorial will guide you through the process of connecting a UBlox GPS
+module to your Leo Rover.
+
+EVK-7 / EVK-8 / EVK-M8 evaluation kits make evaluating the high performance
+u-blox 7, u-blox 8 and u-blox M8 positioning products simple. The built-in USB
+interface provides both power supply and high-speed data transfer, and
+eliminates the need for an external power supply. The u-blox evaluation kits are
+compact, and their user-friendly interface and power supply make them ideally
+suited for use in laboratories, vehicles and outdoor locations.
+
+In this tutorial we will present the process of connecting a UBlox GPS module to
+your Leo Rover and integrating it with the system on the **UBlox EVK-M8N GPS
+module** example.
+
+The steps might slightly differ for other UBlox models, but should be
+essentially similar.
+
+## What to expect?
+
+After completing this tutorial, you should be able to gather the GPS data and
+get it published on ROS topics.
+
+<ImageZoom
+  loading="eager"
+  src="/img/robots/leo/integrations/ublox-gps/ech-ublox-gps-node.webp"
+  width="294"
+  height="431"
+  figStyle={{
+    width: 400,
+  }}
+/>
+
+## Prerequisites
+
+<LinkButton
+  to="/leo-rover/1.8/advanced-guides/ros-development"
+  title="ROS Development"
+  description="Detailed guide on ROS development for Leo Rover, covering topics like adding
+  additional functionalities, building ROS packages and more."
+/>
+
+## Referenced products
+
+<ProductPreview
+  shopUrl="https://www.leorover.tech/shop/gps-module"
+  imageSrc="/img/robots/leo/integrations/ublox-gps/ublox-evk-m8n-shop.webp"
+  width={1200}
+  height={802}
+  title="UBlox EVK-M8N GPS module"
+  description="Enhance Leo Rover's capabilities with the UBlox EVK-M8N GPS module.
+    Experience precise navigation and versatile connectivity with multi-GNSS support,
+    ensuring high-performance and reliability."
+/>
```

Comment:
Move the product preview to `src/products`

Reply:
Ok